zuul.d/jobs.yaml

```diff
@@ -412,152 +412,142 @@
     name: openstacksdk-ansible-devel-functional-devstack
     parent: init-test
     description: |
-      Run openstacksdk ansible functional tests against a master devstack
-      using git devel branch version of ansible.
+      Run openstacksdk ansible functional tests against a master devstack using git devel branch version of ansible.
     run: playbooks/ansible-functional-devstack/run.yaml
     required-projects:
-      - name: theopenlab/ansible
-        override-checkout: devel
+      - theopenlab/ansible
```

Comment:
this job run against project `theopenlab/ansible`, so it can be remove from `required-projects`